apps/example/src/resources/controllers/example.controller.ts

```diff
@@ -1,9 +1,13 @@
 
 import { ExampleService } from "../services/example.service";
-import { InjectService, JsonApi, JsonApiController, JsonApiService, QueryParams } from "nest-json-api";
+import { InjectService, JsonApi, JsonApiController, JsonApiService, QueryParams, ExcludeMethode } from "nest-json-api";
 import { Users } from "database";
 
-@JsonApi(Users)
+@JsonApi(Users,
+  {
+    allowMethode: ExcludeMethode(['getAll', 'deleteRelationship'])
```

Comment:
It is necessary to rename ExcludeMethode function in this example too and it would be better to name function with a lowercase letter: excludeMethod.

Reply:
done